src/borda_clj/client.clj

```diff
@@ -4,26 +4,35 @@
             [byte-streams :as bs]
             [cheshire.core :as json]))
 
-(def discard_key {:op "_discard"})
+(def measure_key {:op "_measure"})
```

Comment:
kebab-case, not snake_case, is idiomatic for lisps.  So, `measure-key` and `measure-succeeded`.

Reply:
Muscle-memory.